catalog_validation/validation.py

```diff
@@ -36,8 +36,11 @@ def validate_train(train_path):
     verrors.check()
 
     for catalog_item in os.listdir(train_path):
+        catalog_item_path = os.path.join(train_path, catalog_item)
+        if catalog_item.startswith('.') or not os.path.isdir(catalog_item_path):
```

Comment:
Out of curiosity, do we need/require directories starting with `.` under a train directory ?

Reply:
ping @Ornias1993